packages/monorepo-split-github-action/entrypoint.sh

```diff
@@ -50,17 +51,22 @@ note "Cleaning destination repository of old files"
 find "$CLONE_DIR" | grep -v "^$CLONE_DIR/\.git" | grep -v "^$CLONE_DIR$" | xargs rm -rf # delete all files (to handle deletions)
 ls -la "$CLONE_DIR"
 
+if test ! -z "$COMMIT_MESSAGE"
+then
+    ORIGIN_COMMIT="https://github.com/$GITHUB_REPOSITORY/commit/$GITHUB_SHA"
+    COMMIT_MESSAGE="${COMMIT_MESSAGE/ORIGIN_COMMIT/$ORIGIN_COMMIT}"
+else
+    COMMIT_MESSAGE=$(git show -s --format=%B "$GITHUB_SHA")
+fi
+
 note "Copying contents to git repo"
 
-cp -r "$PACKAGE_DIRECTORY"/* "$CLONE_DIR"
+cp -Ra $PACKAGE_DIRECTORY/. "$CLONE_DIR"
```

Comment:
What is this for?

Reply:
Comment is needed so the knowledge is in the code for future readers

Reply:
Added.